docker/test/fasttest/run.sh

```diff
@@ -211,6 +211,11 @@ function build
         echo "build_clickhouse_fasttest_binary: [ OK ] $BUILD_SECONDS_ELAPSED sec." \
           | ts '%Y-%m-%d %H:%M:%S' \
           | tee "$FASTTEST_OUTPUT/test_result.txt"
+
+        stripped_clickhouse_shows_symbols_names=FAIL
+        programs/clickhouse-stripped --stacktrace -q 'select' |& grep -q 'DB::LocalServer::main' && stripped_clickhouse_shows_symbols_names=OK
+        echo "stripped_clickhouse_shows_symbols_names: [ $stripped_clickhouse_shows_symbols_names ] 0 sec."
```

Comment:
shouldn't it be written into `test_result.txt`?

Reply:
Ops, forgot this bit, thanks